tools/e2etesting/steps/runtests.yml

```diff
@@ -47,19 +49,19 @@ steps:
 
 - task: AzureKeyVault@1
   displayName: 'Retrieve KeyVault secrets for API tests'
-  condition: eq( '${{ parameters.PublisherMode }}', 'orchestrated')
+  condition: ne( '${{ parameters.ModeValue }}', 'orchestrated')
```

Comment:
I'm not sure if this is correct, we need the secrets from KeyVaault for orchestrated testing

Reply:
Thanks, I had 'standalone' in my mind but forgot to change it